build.sbt

```diff
@@ -35,13 +35,12 @@ val nussknackerV = {
 lazy val root = (project in file("."))
   .enablePlugins(FormatStagedScalaFilesPlugin)
   .aggregate(modules: _*)
-  .settings(commonSettings(scala212V))
-  .settings(
-    name := "nussknacker-flink-compatibility"
-  )
+  .settings(commonSettings)
+  .settings(disablePublish)
+  .settings(name := "nussknacker-flink-compatibility")
 
-def commonSettings(scalaV: String) =
-  Seq(
+lazy val commonSettings =
+  publishSettings ++ Seq(
```

Comment:
we add the `publishSettings` to `commonSettings`, and then `disablePublish` in some modules. Does it mean that `publishSettings` is not really a common setting? ;)